datasets/kor_qpair/kor_qpair.py

```diff
@@ -0,0 +1,103 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Korean pairwise question binary classification dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+_CITATION = """\
+@misc{Song:2018,
+  title     = "Paired Question v.2",
+  authors   = "Youngsook Song",
+  publisher = "GitHub",
+  year      = "2018"
+}
+"""
+
+
+_DESCRIPTION = """\
+This is a Korean paired question dataset containing labels indicating whether two questions in a given pair are semantically identical. This dataset was used to evaluate the performance of [KoGPT2](https://github.com/SKT-AI/KoGPT2#subtask-evaluations) on a phrase detection downstream task.
+"""
+
+
+_HOMEPAGE = "https://github.com/songys/Question_pair"
+
+_LICENSE = "The MIT License (MIT)"
+
+_URL = "https://raw.githubusercontent.com/songys/Question_pair/master/"
+_URLs = {key: f"{_URL}{key}.txt" for key in ("train", "test", "validation")}
+
+
+class KorQPair(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class KorQpair(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `kor_qpair`